app/src/ui/diff/image-diffs/two-up.tsx

```diff
@@ -27,40 +29,69 @@ export class TwoUp extends React.Component<ITwoUpProps, {}> {
     const previousImageSize = this.props.previousImageSize || zeroSize
     const currentImageSize = this.props.currentImageSize || zeroSize
+    const diffPercent = percentDiff(
+      this.props.previous.bytes,
+      this.props.current.bytes
+    )
+    const diffBytes = this.props.current.bytes - this.props.previous.bytes
 
     return (
-      <div className="image-diff-two-up" ref={this.props.onContainerRef}>
-        <div className="image-diff-previous">
-          <div className="image-diff-header">Deleted</div>
-          <ImageContainer
-            image={this.props.previous}
-            onElementLoad={this.props.onPreviousImageLoad}
-            style={style}
-          />
+      <div className="image-diff-container" ref={this.props.onContainerRef}>
+        <div className="image-diff-two-up">
+          <div className="image-diff-previous">
+            <div className="image-diff-header">Deleted</div>
+            <ImageContainer
+              image={this.props.previous}
+              onElementLoad={this.props.onPreviousImageLoad}
+              style={style}
+            />
 
-          <div className="image-diff-footer">
-            <span className="strong">W:</span> {previousImageSize.width}px |{' '}
-            <span className="strong">H:</span> {previousImageSize.height}px
+            <div className="image-diff-footer">
+              <span className="strong">W:</span> {previousImageSize.width}px |{' '}
+              <span className="strong">H:</span> {previousImageSize.height}px |{' '}
+              <span className="strong">Size:</span>{' '}
+              {formatBytes(this.props.previous.bytes)}
+            </div>
           </div>
-        </div>
 
-        <div className="image-diff-current">
-          <div className="image-diff-header">Added</div>
-          <ImageContainer
-            image={this.props.current}
-            onElementLoad={this.props.onCurrentImageLoad}
-            style={style}
-          />
+          <div className="image-diff-current">
+            <div className="image-diff-header">Added</div>
+            <ImageContainer
+              image={this.props.current}
+              onElementLoad={this.props.onCurrentImageLoad}
+              style={style}
+            />
 
-          <div className="image-diff-footer">
-            <span className="strong">W:</span> {currentImageSize.width}px |{' '}
-            <span className="strong">H:</span> {currentImageSize.height}px
+            <div className="image-diff-footer">
+              <span className="strong">W:</span> {currentImageSize.width}px |{' '}
+              <span className="strong">H:</span> {currentImageSize.height}px |{' '}
+              <span className="strong">Size:</span>{' '}
+              {formatBytes(this.props.current.bytes)}
+            </div>
           </div>
         </div>
+        <div className="image-diff-summary">
+          Diff:{' '}
+          <span
+            className={classNames({
```

Comment:
Should this logic be moved outside of `return()`? Only asking since I'm not familiar with this pattern.

Reply:
No, that looks reasonable to me.

Reply:
looks fine to me 👍